app/presenters/base.presenter.js

```diff
@@ -42,6 +42,21 @@ class BasePresenter {
       .toString()
       .padStart(length, '0')
   }
+
+  /**
+   * If the value passed in is a credit then returns the value as a negative; otherwise returns it as a positive
+   */
+  _signedCreditValue (value, credit) {
+    return credit ? -value : value
+  }
```

Comment:
I think this could be simplified and the intent made clearer by just using some math and essentially removing the word 'credit' from the name/

```suggestion
  _reverseSignOfValue (value) {
    return value * -1
  }
```